boms/geode-all-bom/src/test/resources/expected-pom.xml

```diff
@@ -67,6 +67,12 @@
         <version>2.9.8</version>
         <scope>compile</scope>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
```

Comment:
Can you point me where this new dependency is used in the test? I don't see it is used in the tests of this pull request.

Reply:
The test uses a java.time.LocalTime as one of the values in the domain object to show it will be displayed correctly.